composer.json

```diff
@@ -11,13 +11,13 @@
         }
     ],
     "require": {
-        "php": ">=7.2.0",
+        "php": ">=7.3",
```

Comment:
can we have 7.4 since we don't have it 7.3 in ci?

Reply:
Yeah but I wanted to copy this: https://github.com/GetStream/stream-php/blob/main/composer.json#L14

Although in the CI, I was thinking about testing with the latest 7.* and latest 8.*

Reply:
so let's add this too https://github.com/GetStream/stream-php/blob/main/.github/workflows/ci.yml#L11

Reply:
✅